toolchains/esp32c6/sdkconfig.defaults

```diff
@@ -34,7 +35,11 @@ CONFIG_ESP_TASK_WDT_INIT=n
 CONFIG_ESP_WIFI_DYNAMIC_RX_BUFFER_NUM=16
 CONFIG_ESP_WIFI_DYNAMIC_TX_BUFFER_NUM=16
 CONFIG_ESP_WIFI_NVS_ENABLED=n
+CONFIG_ESP_WIFI_SLP_IRAM_OPT=n
 CONFIG_ESP_WIFI_GMAC_SUPPORT=n
+CONFIG_ESP_WIFI_ENABLE_WIFI_TX_STATS=y
```

Comment:
Necessary?

Reply:
I will remove it in a follow-up PR.
This PR only updates to the newer esp-idf.